frontend/src/App.js

```diff
@@ -1,12 +1,32 @@
+import AppHeader from './components/AppHeader'
+import { Route, Switch } from 'react-router-dom'
+import Landing from './pages/Landing'
+import ArtistSearchResult from './pages/ArtistSearchResult'
+import styled from 'styled-components/macro'
 
-function App() {
+export default function App() {
   return (
-    <div>
-      <header>
-          Welcome to musiCompass!
-      </header>
-    </div>
-  );
+    <PageLayout>
+      <AppHeader/>
+      <Switch>
+        <Route exact path="/">
+          <Landing />
```

Comment:
you may want to consider a more meaningful name for your component, e.g. Homepage, because Landing could be anything

Reply:
as far as I see it homepage and landing(page) are interchangable